introduction.md

```diff
@@ -0,0 +1 @@
+Hello My name is Jean-Marcel Belmont and I am a code school fan and student.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.